include/linux/mod_devicetable.h

```diff
@@ -249,6 +249,8 @@ struct hda_device_id {
 };
 
 struct sdw_device_id {
+	__u8  class_id;
+	__u8  sdw_version;
```

Comment:
nick-pick:  class_id and sdw_version are defined before mfg_id & part_id, but they are used after mfg_id & part_id in the following changes.

Reply:
reordered now, thanks